rotkehlchen/chain/manager.py

```diff
@@ -114,14 +116,18 @@ class BlockchainBalances:
     db: DBHandler  # Need this to serialize BTC accounts with xpub mappings
     eth: DefaultDict[ChecksumEthAddress, BalanceSheet] = field(init=False)
     btc: Dict[BTCAddress, Balance] = field(init=False)
+    # TODO: consider at some point support BalanceSheet
+    ksm: Dict[KusamaAddress, Balance] = field(init=False)
```

Comment:
I understand that at the moment we go with Balance, no?

Reply:
Done. Here some DeFi protocols for Polkadot. It would made sense to me to have them tested on Kusama (another thing is the economical relevance of liabilities in a testing network). Just in case I've added them and got familiarised with the logic, so for Polkadot I will already know.

https://www.parity.io/defi-on-polkadot-an-ecosystem-overview/